src/handlers/PinchGestureHandler.ts

```diff
@@ -0,0 +1,26 @@
+import createHandler from './createHandler';
+import {
+  BaseGestureHandlerProps,
+  baseGestureHandlerProps,
+} from './gestureHandlerCommon';
+
+export type PinchGestureHandlerEventPayload = {
+  scale: number;
+  focalX: number;
+  focalY: number;
+  velocity: number;
+};
+
+export interface PinchGestureHandlerProps
+  extends BaseGestureHandlerProps<PinchGestureHandlerEventPayload> {}
+
+export type PinchGestureHandler = typeof PinchGestureHandler;
+// eslint-disable-next-line @typescript-eslint/no-redeclare -- backward compatibility; see description on the top of this file
```

Comment:
And here.

Reply:
Fixed in 4f58a3ff228fde7095fefb8cfec8843288fcb2af